packages/react/package.json

```diff
@@ -52,7 +52,7 @@
     "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react@1.2.9",
     "@radix-ui/react-accordion": "^0.1.1",
     "@radix-ui/react-dropdown-menu": "^0.1.1",
-    "@radix-ui/react-id": "^0.1.0",
+    "@radix-ui/react-id": "0.1.1",
     "@radix-ui/react-slider": "^0.1.1",
     "@radix-ui/react-tabs": "^0.1.1",
```

Comment:
It may be best to pin _all_ `@radix-ui/*` dependencies given today's scenario.

Reply:
good call, preparing another commit

Reply:
done